### PR TITLE
doc: subcommand workflows

### PR DESCRIPTION
**Pull request by itsspriyansh**
This PR adds the usage instructions for the `connect`, `install`, `uninstall` and `list` android subcommands

### CODE REVIEW

**Comment by itsspriyansh** [comment]
Currently adding docs only for the scripts which have been merged into main.

File: docs/subcommands/connect.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Workflows of the `connect` subcommand
+
+**Syntax**
+```sh
+npx @nightwatch/mobile-helper android connect [flags|cliConfigs]
```
Comment (garg3133) [inline docs/subcommands/connect.md:5]:
```suggestion
npx @nightwatch/mobile-helper android connect FLAG [configs]
```

File: docs/subcommands/install.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Workflows of the `install` subcommand
+
+**Syntax**
+```sh
+npx @nightwatch/mobile-helper android install [flags|cliConfigs]
```
Comment (garg3133) [inline docs/subcommands/install.md:5]:
```suggestion
npx @nightwatch/mobile-helper android install FLAG [configs]
```

File: docs/subcommands/install.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+### 1. Install an APK
+
+Run the below command to install an APK on a real device or an AVD:
+```sh
+npx @nightwatch/mobile-helper android install --app [cliConfigs]
+```
```
Comment (garg3133) [inline docs/subcommands/install.md:13]:
```suggestion
Run the below command to install an APK on a real device or an AVD:

```sh
npx @nightwatch/mobile-helper android install --app

# with configs
npx @nightwatch/mobile-helper android install --app [--deviceId <device_id>] [--path <path_to_apk>]
```
```

File: docs/subcommands/uninstall.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+## Workflows of the `uninstall` subcommand
+
+**Syntax**
+```sh
+npx @nightwatch/mobile-helper android uninstall [flags|cliConfigs]
```
Comment (garg3133) [inline docs/subcommands/uninstall.md:5]:
```suggestion
npx @nightwatch/mobile-helper android uninstall FLAG [configs]
```

File: docs/subcommands/uninstall.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+```
+
+### 1. Delete an Android Virtual Device
+
+Run the below command to delete a currently present AVD:
```
Comment (garg3133) [inline docs/subcommands/uninstall.md:10]:
```suggestion
Run the below command to delete an AVD:
```

File: docs/subcommands/install.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### 1. Install an APK
+
+Run the below command to install an APK on a real device or an AVD:
+
+```sh
+npx @nightwatch/mobile-helper android install --app
+
+# with configs
+npx @nightwatch/mobile-helper android install --app [--deviceId <device_id>] [--path <path_to_apk>]
```
Comment (garg3133) [inline docs/subcommands/install.md:16]:
```suggestion
npx @nightwatch/mobile-helper android install --app [--deviceId <device_id>] [--path <path_to_apk>]
```
```

**Pull request merged**